src/main/resources/bootstrap.yaml

```diff
@@ -15,7 +15,7 @@ spring:
         idam-s2s-api: idam.s2s-auth.url
         sscs-s2s-secret: idam.s2s-auth.totp_secret
         s2s-micro: idam.s2s-auth.microservice
-        sscs-asb-primary-send-and-listen-shared-access-key: amqp.password
-        docmosis-api-key: pdf-service.accessKey
-        notification-key: gov.uk.notification.api.key
-        notification-test-key: gov.uk.notification.api.testKey
\ No newline at end of file
+        sscs-asb-primary-send-and-listen-shared-access-key: AMQP_PASSWORD
```

Comment:
should we be using ENV's or config values from the application yaml values in this file?

Reply:
that does not look right to me either but I might be missing something

Reply:
Discussing with @timja, I think we can use either, unless the variable clashes with Spring, in which case we should use the a.b.c notation

Reply:
Given your application.yaml, I prefer this way as I find it easier to follow. It can work also the other way, depending on what you have in your environment.